packages/wallet-adapter-core/src/utils/helpers.ts

```diff
@@ -0,0 +1,26 @@
+export function isMobile(): boolean {
+  return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
```

Comment:
FYI: this might be good too https://gist.github.com/dalethedeveloper/1503252#the-98-solution, I'm amazed no one really has made a standard solution for this